### ISSUE TITLE
NBench.Runner.DotNetCli can't run benchmarks that have external dependencies

### ISSUE THREAD

**Issue by heynickc** [petabridge/NBench#192]
For example, running benchmarks that pull in Akka.NET, like Akka.Tests.Performance, throws the following error:

```
ERROR: Error while executing the tests.
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.RuntimeAssembly.get_DefinedTypes()
   at NBench.Sdk.Compiler.ReflectionDiscovery.ClassesWithPerformanceBenchmarks(Assembly targetAssembly) in
...
```
Looking deeper, it will show that it cannot load the Akka.dll assembly, even though it is in the bin folder with the benchmark assembly.

**Comment by Aaronontheweb** [comment]
@heynickc how close to a "work in progress" PR on this?

**Comment by heynickc** [comment]
@Aaronontheweb #193.  It swallows an exception, but it runs now against Akka.Tests.Performance.dll

**Comment by heynickc** [comment]
Closed by #198
